src/mame/mame.lst

```diff
@@ -32779,6 +32779,9 @@ tx0_64kw                        // April 1956 MIT TX-0 (64kw RAM)
 tx0_8kwo                        // 1959 MIT TX-0 (8kw RAM)
 tx0_8kw                         // 1962 MIT TX-0 (8kw RAM)
 
+@source:mit/cadr.cpp
+cadr
+
 @source:mits/altair.cpp
```

Comment:
Please keep this file sorted – “c” comes before “t”.  Also I think you’ll get merge conflicts from the comments being stripped.